content/en/news/releases/1.7.x/announcing-1.7.6/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Announcing Istio 1.7.6
+linktitle: 1.7.6
+subtitle: Patch Release
+description: Istio 1.7.6 patch release.
+publishdate: 2020-12-10
+release: 1.7.6
+aliases:
+- /news/announcing-1.7.6
+---
+
+This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.7.5 and Istio 1.7.6
+
+{{< relnote >}}
+
+## Changes
+
+- **Fixed** Fixed an issue causing telemetry HPA settings to be overridden by the inline replicas. ([Issue #28916](https://github.com/istio/istio/issues/28916))
```

Comment:
Remove the redundant Fixed at the start of each line. `**Fixed** Fixed an` -> `** Fixed** an`